cfme/tests/infrastructure/test_timelines.py

```diff
@@ -154,49 +236,107 @@ def _check_timelines(self, target):
         if targets:
             for target in targets:
                 try:
-                    wait_for(self._check_timelines, [target], timeout='5m', fail_condition=0)
+                    wait_for(self._check_timelines,
+                             [target, policy_events],
+                             timeout='7m',
+                             fail_condition=0)
                 except TimedOutError:
                     soft_assert(False, '0 occurrence of {} found on the timeline of {}'.format(
                         self.event, target))
         else:
             raise ValueError('Targets must not be empty')
 
 
-def test_timeline_events(new_vm, soft_assert):
-    events_list = ['create', 'stop', 'start', 'suspend', 'start']
+def test_infra_timeline_create_event(new_vm, soft_assert):
+    vm_event = VMEvent(new_vm, 'create')
     targets = (new_vm, new_vm.cluster, new_vm.host, new_vm.provider)
-    for event in events_list:
-        vm_event = VMEvent(new_vm, event)
-        logger.info('Will generate event %r on machine %r', event, new_vm.name)
-        wait_for(vm_event.emit, timeout='5m', message='Event {} did timeout'.format(event))
-        vm_event.catch_in_timelines(soft_assert, targets)
+    logger.info('Will generate event "create" on machine %r', new_vm.name)
+    wait_for(vm_event.emit, timeout='7m', message='Event "create" failed')
+    vm_event.catch_in_timelines(soft_assert, targets)
+
 
+@pytest.mark.meta(blockers=[BZ(1542962, forced_streams=['5.8'])])
+def test_infra_timeline_policy_event(new_vm, control_policy, soft_assert):
+    event = 'policy'
+    targets = (new_vm, new_vm.host, new_vm.cluster, new_vm.provider)
+    vm_event = VMEvent(new_vm, event)
+    wait_for(vm_event.emit, timeout='7m', message='Event {} did timeout'.format(event))
+    vm_event.catch_in_timelines(soft_assert, targets, policy_events=True)
 
-def test_infra_diagnostic_timeline_events(new_vm, soft_assert, mark_vm_as_appliance):
-    events_list = ['create']
+
+def test_infra_timeline_diagnostic(new_vm, soft_assert, mark_vm_as_appliance):
+    event = 'create'
     targets = (new_vm.appliance.server,)
+    logger.info('Will generate event %r on machine %r', event, new_vm.name)
+    vm_event = VMEvent(new_vm, event)
+    vm_event.catch_in_timelines(soft_assert, targets)
+
+
+def test_infra_timeline_power_events(new_vm, soft_assert):
+    events_list = ['stop', 'start', 'suspend', 'start']
+    targets = (new_vm, new_vm.cluster, new_vm.host, new_vm.provider)
     for event in events_list:
-        logger.info('Will generate event %r on machine %r', event, new_vm.name)
         vm_event = VMEvent(new_vm, event)
+        logger.info('Will generate event %r on machine %r', event, new_vm.name)
+        wait_for(vm_event.emit, timeout='7m', message='Event {} failed'.format(event))
         vm_event.catch_in_timelines(soft_assert, targets)
+
+
+def test_infra_timeline_rename_event(new_vm, soft_assert):
```

Comment:
Next 4 test can be done as one parametrized test

Reply:
No because there is a bug in pytest that make parametrized  not working when defining a fixture with a module scope, they are falling to function scope.